package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cordova-electron",
-  "version": "1.1.0-dev",
+  "version": "1.0.2",
```

Comment:
this seems wrong

Reply:
Yeah, not sure why it changed.

Reply:
Removed the change and is now back to `1.1.0-dev`